sklearn/kernel_approximation.py

```diff
@@ -670,14 +670,16 @@ class Nystroem(TransformerMixin, BaseEstimator):
         Pass an int for reproducible output across multiple function calls.
         See :term:`Glossary <random_state>`.
 
+    n_jobs : int, default=None
+        The number of jobs to use for the computation. This works by breaking
+        down the kernel matrix into n_jobs even slices and computing them in
+        parallel.
+
     Attributes
     ----------
     components_ : ndarray of shape (n_components, n_features)
         Subset of training points used to construct the feature map.
 
-    component_indices_ : ndarray of shape (n_components)
-        Indices of ``components_`` in the training set.
-
```

Comment:
But why would remove this attribute in the first place? This can be useful for model inspection and this is kind of unrelated to this PR.

Reply:
I just can't see any advantage of `component_indices_` if we already have `components_`. This should just be a proposal for simplification but is nothing necessary.

Reply:
This is a change in public API so this would have to go through a deprecation phase. Also I still believe that this is a nice convenience to have access to the indices of the samples in case the features have been heavily transformed via a pipeline of transformers for instance.

In any case it's better to make pull requests focused on a specific topic.